### PR TITLE
Update bridge tutorial

### PR DESCRIPTION
**Pull request by TucksonDev**
This PR updates the Bridge tutorial:
- Adds the notion of bridging from a parent chain to a child chain (with examples for new users)
- Adapts the contents to follow standard pattern guides
- Prioritizes Arbitrum Sepolia over Arbitrum Goerli

[Preview](https://nitro-docs-git-update-bridge-tutorial-offchain-labs.vercel.app/getting-started-users)

### CODE REVIEW

File: arbitrum-docs/getting-started-users.mdx [hunk 1]
```diff
@@ -1,110 +1,105 @@
...
 
-#### 1- Add a Network
+There are several ways to obtain the native currency:
 
-To start using the Arbitrum bridge, you need to add the desired chain's RPC endpoint to your wallet. Here, we provide an example for doing this using the MetaMask wallet.
-You need to first click on the MetaMask extension on your browser, click MetaMask's network selector dropdown, and then click the `Add Network` button.
-Click "Add a network manually" and then provide the information corresponding to the Arbitrum chain you want to connect to:
+- Using a [supported centralized exchange](https://portal.arbitrum.io/?categories=centralized-exchanges), which allows you to purchase ETH and withdraw it to your wallet.
+- Using an [on-ramp service](https://portal.arbitrum.io/?categories=fiat-on-ramp), which allows you to purchase ETH and send it directly to your wallet.
+- If you are using a testnet, requesting funds from a faucet for [Sepolia](https://sepoliafaucet.com/), [Goerli](https://goerlifaucet.com) or [Arbitrum Sepolia](https://faucet.quicknode.com/arbitrum/sepolia)
```
Comment (mahsamoosavi) [inline arbitrum-docs/getting-started-users.mdx:27]:
```suggestion
- If you are using a testnet, requesting funds from a faucet for [Sepolia](https://sepoliafaucet.com/), [Goerli](https://goerlifaucet.com) or [Arbitrum Sepolia](https://faucet.quicknode.com/arbitrum/sepolia).
```

File: arbitrum-docs/getting-started-users.mdx [hunk 2]
```diff
@@ -1,110 +1,105 @@
...
 
-![Add the desired Arbitrum Network to your MetaMask wallet](assets/getting_started_users_1.png)
+### Step 2: Add the network to your wallet
```
Comment (mahsamoosavi) [inline arbitrum-docs/getting-started-users.mdx:29]: should we say "Add the preferred network to your wallet"?

File: arbitrum-docs/getting-started-users.mdx [hunk 3]
```diff
@@ -1,110 +1,105 @@
...
 
-##### Arbitrum Nova:
+### Step 3: Initiate the deposit
 
-- Network Name: `Arbitrum Nova`
-- New RPC URL: `https://nova.arbitrum.io/rpc`
-- Chain ID: `42170`
-- Currency Symbol: `ETH`
-- Block Explorer URL: `https://nova.arbiscan.io/`
+To bridge your ETH or ERC-20 tokens to a different chain, start by visiting [bridge.arbitrum.io](https://bridge.arbitrum.io/). Login to the bridge with your wallet and make sure you are connected to the source network (from where you want to deposit your assets) at the top of the page. Then, select the destination network (where you want your assets to go), e.g., Arbitrum One or Arbitrum Nova.
```
Comment (mahsamoosavi) [inline arbitrum-docs/getting-started-users.mdx:47]:
```suggestion
To bridge your ETH or ERC-20 tokens to a different chain, start by visiting [bridge.arbitrum.io](https://bridge.arbitrum.io/). Log in to the bridge with your wallet and make sure you are connected to the source network (from where you want to deposit your assets) at the top of the page. Then, select the destination network (where you want your assets to go), e.g., Arbitrum One or Arbitrum Nova.
```

File: arbitrum-docs/getting-started-users.mdx [hunk 4]
```diff
@@ -1,110 +1,105 @@
...
 
-A countdown will pop up stating you’ll get your funds in 7-8 days.
+A countdown will pop up stating that you’ll get your funds in 7-8 days.
 
 You can check the status of your withdrawal by clicking on your profile on the top right and opening the `Transactions` tab, and claim it there when it’s ready.
 
 ![See the transaction history](assets/getting_started_users_7.png)
 
-Once the countdown is done, switch to the Ethereum mainnet network on your wallet and press the `claim` button that has now turned blue to receive your funds!
+Once the countdown is done, switch to the destination network on your wallet and press the `Claim` button that has now turned blue to receive your funds!
```
Comment (mahsamoosavi) [inline arbitrum-docs/getting-started-users.mdx:101]:
```suggestion
Once the countdown is done, switch to the destination network on your wallet and press the `Claim` button, that has now turned blue, to receive your funds!
```

File: arbitrum-docs/getting-started-users.mdx [hunk 5]
```diff
@@ -1,110 +1,105 @@
...
 
-#### What's Next?
+## What's Next?
```
Comment (mahsamoosavi) [inline arbitrum-docs/getting-started-users.mdx:103]:
```suggestion
## What's next?
```

**Review by mahsamoosavi — CHANGES_REQUESTED** [review]

**Review by mahsamoosavi — APPROVED** [review]

**Review by symbolpunk — APPROVED** [review]
BAMF

**Pull request merged**